src/server/input_service.rs

```diff
@@ -1609,6 +1638,7 @@ pub fn handle_key_(evt: &KeyEvent) {
     #[cfg(not(any(target_os = "android", target_os = "ios")))]
     let mut _lock_mode_handler = None;
     #[cfg(not(any(target_os = "android", target_os = "ios")))]
+    if evt.down {
```

Comment:
100% sure?

Reply:
Yes, I've tested again. Windows, Linux and MacOS as the controlled side, "Map mode" and "Translate mode".